README.md

```
@@ -1,53 +1,223 @@
+# Run specific package tests
+go test ./internal/parse
+go test ./internal/confluence
+go test ./internal/bitbucket
+
+# Run with verbose output
+go test -v ./...
+```
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)